Controller/MediaAdminController.php

```diff
@@ -64,7 +65,7 @@ public function browserAction()
         $formView = $datagrid->getForm()->createView();
 
         // set the theme for the current Admin Form
-        $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
+        $this->setFormTheme($formView, $this->admin->getFilterTheme()[0]);
```

Comment:
`$this->admin->getFilterTheme()[0]` is a PHP 5.4+ feature. Our minimum requirement is PHP 5.3.3. But I don't understand why you pass the first index instead of all the array here...